BeagleBone_black/src/sensors/vl6180.cpp

```diff
@@ -154,6 +154,23 @@ uint8_t VL6180::getDistance()
   }
 }
 
+bool VL6180::isOnline()
+{
+  uint8_t data;
+  uint8_t status;
+  uint8_t distance;
+
+  readByte(kResultRangeStatus, &data);
+  status = data >> 4;
+  distance = getDistance();
+
+  if (status == 0 && distance != 0 && distance != 255) {
```

Comment:
this method should only check if we have a i2c connection to the sensor. This should be equivalent to successful read of the WHOAMI register. Data health check should be performed in the `getData` method in which the health should be indicated by setting the `proxi->operational` flag.